src/core/cgroup.c

```diff
@@ -3054,12 +3057,16 @@ int unit_check_oomd_kill(Unit *u) {
         if (!increased)
                 return 0;
 
-        if (n > 0)
+        if (n > 0) {
                 log_unit_struct(u, LOG_NOTICE,
                                 "MESSAGE_ID=" SD_MESSAGE_UNIT_OOMD_KILL_STR,
                                 LOG_UNIT_INVOCATION_ID(u),
                                 LOG_UNIT_MESSAGE(u, "systemd-oomd killed %"PRIu64" process(es) in this unit.", n));
 
+                if (UNIT_VTABLE(u)->notify_cgroup_oom)
+                        UNIT_VTABLE(u)->notify_cgroup_oom(u, true);
```

Comment:
in new code when we call a function that takes a boolean flag style argument we prefer if a comment is added giving a hint what this parameter is about, i.e.:

```c
UNIT_VTABLE(u)->notify_cgroup_oom(u, /* managed= */ true);
```